service/oceancd/verificationProvider.go

```diff
@@ -0,0 +1,510 @@
+	VerificationProvider *VerificationProvider `json:"verificationProvider,omitempty"`
+}
+
+type PatchVerificationProviderOutput struct {
+	VerificationProvider *VerificationProvider `json:"verificationProvider,omitempty"`
+}
+
+type DeleteVerificationProviderInput struct {
+	Name *string `json:"name,omitempty"`
+}
+
+type DeleteVerificationProviderOutput struct{}
+
+func verificationProviderFromJSON(in []byte) (*VerificationProvider, error) {
+	b := new(VerificationProvider)
+	if err := json.Unmarshal(in, b); err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+func verificationProvidersFromJSON(in []byte) ([]*VerificationProvider, error) {
+	var rw client.Response
+	if err := json.Unmarshal(in, &rw); err != nil {
+		return nil, err
+	}
+	out := make([]*VerificationProvider, len(rw.Response.Items))
+	if len(out) == 0 {
+		return out, nil
+	}
+	for i, rb := range rw.Response.Items {
+		b, err := verificationProviderFromJSON(rb)
+		if err != nil {
+			return nil, err
+		}
+		out[i] = b
+	}
+	return out, nil
+}
+
+func verificationProvidersFromHttpResponse(resp *http.Response) ([]*VerificationProvider, error) {
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	return verificationProvidersFromJSON(body)
+}
+
+// endregion
+
+// region API requests
+
+func (s *ServiceOp) ListVerificationProviders(ctx context.Context) (*ListVerificationProvidersOutput, error) {
+	r := client.NewRequest(http.MethodGet, "/ocean/cd/verificationProvider")
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &ListVerificationProvidersOutput{VerificationProviders: gs}, nil
+}
+
+func (s *ServiceOp) CreateVerificationProvider(ctx context.Context, input *CreateVerificationProviderInput) (*CreateVerificationProviderOutput, error) {
+	r := client.NewRequest(http.MethodPost, "/ocean/cd/verificationProvider")
+	r.Obj = input
+
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	output := new(CreateVerificationProviderOutput)
+	if len(gs) > 0 {
+		output.VerificationProvider = gs[0]
+	}
+
+	return output, nil
+}
+
+func (s *ServiceOp) ReadVerificationProvider(ctx context.Context, input *ReadVerificationProviderInput) (*ReadVerificationProviderOutput, error) {
+	path, err := uritemplates.Expand("/ocean/cd/verificationProvider/{name}", uritemplates.Values{
+		"name": spotinst.StringValue(input.Name),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	r := client.NewRequest(http.MethodGet, path)
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
```

Comment:
use meaningful name for variable instead of gs

Reply:
changed to vp